src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
```

Comment:
The enum declaration looks good and follows the convention of naming enum constants in uppercase. Make sure to use the `name()` method when you need to get the String representation of these enum constants.